### DISCUSSION TITLE
Getting accepted as an outreachy intern

### DISCUSSION THREAD

**Discussion by mina-atef-00** [LesFruitsDefendus/saskatoon-ng#295] [Outreachy]
I am very very thankful for Outreachy and Les Fruits Defendus for providing me this great opportunity. And **especially** my mentors @tristanlatr and @2ynn for helping me greatly through those two months and entrusting me with this position.

I hope this internship will be very productive and that we will make the website more robust and easily usable for the volunteers at Les Fruits Defendus.

Thank you very much!

**Comment by tristanlatr** [comment]
Great to have you in our team !

Talk to you later,

**Comment by mina-atef-00** [comment]
Hello @tristanlatr @2ynn ,
I have some questions regarding the start of the internship. How can I contact you privately?

**Reply by tristanlatr** [reply]
Hello @mina-andajos, 

We sent you an email a few days ago, I’m in cc. So you can send me an email. 

Talk to you later,
